docs-src/quickstart.rst

```diff
@@ -840,3 +840,11 @@ implementation of :py:class:`genome_kit.data_manager.DataManager` and register i
     # endpoint under the "genomekit.plugins.data_manager" group. GenomeKit will automatically
     # use the plugin's data manager.
     # (see https://setuptools.pypa.io/en/latest/userguide/entry_point.html#entry-points-for-plugins)
+
+
+Logging and troubleshooting
+----------------------
```

Comment:
is there a doc build in the OSS repo?  this will likely throw warnings

Reply:
Limited to release-please publish tags.
I ran `make html` locally and didn't get any errors. 🤷

Reply:
I get something different:

```
Logging and troubleshooting
----------------------
/Users/steve/work/a/docs-src/quickstart.rst:846: WARNING: Title underline too short.
```

maybe we can build but don't publish (like the internal builds)?

Reply:
Oh, a warning. I missed that. It doesn't fail the make command, so how would we detect it?

Reply:
I've only caught it manually, didn't put any time to automate it

Reply:
as an aside, there's still this C++ warning: https://github.com/deepgenomics/GenomeKit/pull/44#issue-2236637291

Reply:
just a drive by but can you also fix the
```
Jetbrains CLion setup
--------------
```
warning?

Reply:
Fixed all doc warnings (found a couple headers that weren't being shown).

I'll try to add checks so that PRs are flagged when there are docs warnings.